index.html

```diff
@@ -109,6 +107,162 @@ <h5><strong>Skills:</strong></h5>
+                <a href="#" class="butn"><button type="button">See Project</button></a>
+              </div>
+            </div>
+
+            <!-- work-4 -->
+            <div>
+              <div class="image">
+                <img src="./images/multi-post.jpg" alt="Recent Projetcs" style="width: 100%;">
+              </div>
+              <div class="multi-post">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalised
+                   reads; no accounts or sign-ups required, has
+                   been the industry's standard dummy text ever
+                   since the 1500s, when an unknown printer took
+                   a standard dummy text.
+                </p>
+                <ul class="specialize">
+                  <li>css</li>
+                  <span class="trim-line"></span>
+                  <li>html</li>
+                  <span class="trim-line"></span>
+                  <li>bootstrap</li>
+                  <span class="trim-line"></span>
+                  <li>Ruby</li>
+                </ul>
+                <a href="#" class="butn"><button type="button">See Project</button></a>
+              </div>
+            </div>
+          </section>
+
+          <!-- About-section-->
+          <section id="about" class="abt-me">
+            <div class="post-about">
+              
+              <div class="multi-post">
+                <h3>Multi-Post Stories</h3>
+                <div class="img">
+                  <img src="./images/Snapshoot Portfolio.png" alt="Recent Projetcs" style="width: 100%;">
+                </div>
+                <p class="mult-txt">Lorem Ipsum is simply dummy text
+                  of the printing and typesetting industry
+                  Lorem ipsum has been the industry's dummy
+                  text ever since 1500s, when an unknown 
+                  printer took the gallery of type and
+                  scrambled it since 1960s with the relesLorem
+                  ipsum is simply dummy text of the printing
+                  and typesetting ever since the 1500s, when an 
+                  unknown printer took a gallery of type veris
+                  lapao todoe.
+                </p>
+                <ul class="specia">
+                  <span class="vert-line"></span>
+                  <li>html</li>
+                  <span class="vert-line"></span>
+                  <li>Ruby on rails</li>
+                  <span class="vert-line"></span>
+                  <li>css</li>
+                  <span class="vert-line"></span>
+                </ul>
+
+                <div class="abt-btns">
+                  <button type="button" class="butn btn">
+                    See Live  <a href="#" class="fa fa-power-off"></a>
+                  </button>
+                  <button type="button" class="butn btnl">
+                    See Source  <a href="#" class="fa fa-github"></a>
+                  </button>
```

Comment:
- [ ] In your design for the last project you have two buttons but in Figma design, there is only one for the `See project`. Also, change the color of the button :upside_down_face: :upside_down_face:
Your design
![Screenshot from 2022-07-02 13-48-32](https://user-images.githubusercontent.com/74501523/176999836-47782076-13a4-436a-8bdf-2daee31ec582.png)

Figma design
![Screenshot from 2022-07-02 13-49-41](https://user-images.githubusercontent.com/74501523/176999852-80787351-8f25-454f-987c-aaa321db4723.png)

Reply:
Thanks a lot for the highlight, according to the figma template design that I chose (template 4) there are two buttons at the end of the About section ( [See Live] & [See Source] buttons), unlike the first template which was used as the example. Kindly, if you can check with figma template 4. Thank you once more

Reply:
Ok my bad, sorry it was my mistake by the way, I instead included the mobile detail section